docs/src/08_concepts.adoc

```diff
@@ -164,15 +337,19 @@ flowchart LR
 ....
 
 ==== Session handling
-*TBD*: talk about JWT token found in auth-service.js
 
-=== Development concepts
+To handle the different sessions of our different users, JWT are also used
+for this purpose. For each logged in user, a token will be generated for
+authenticating them as stated previosly, and also, for mantaining a session
+and hold the necessary information without having to reenter the users' credentials.
+In this way, the user will have to keep on its browser this token to maintain
+alive the session.
 
-==== Testing
-*TBD*
+=== Operation concepts - TBD
 
-- Talk about "super-test" dependency to test expresJS
-- JEST
-- Test automation by Github Actions
+==== Monitoring with Graphana & Promotheus
+Explained in labs after 2nd delivery
 
-Could add more in the future...Refer to https://docs.arc42.org/section-8/
\ No newline at end of file
+==== Peristing the Persistance 
```

Comment:
Is this a typo?

Reply:
first of all, yeah, it should be "Persisting the Persistance".
second of all, it is like a momentary "joke" since for the next sprint we'll have to think a way to keep (persist) the Volumes generated by Docker wich holds all the data used by our application (the persistance).

I just left there as a note that we'll have to tackle that later on.